### PR TITLE
TASK: Add translation IDs for placeholders

### PR DESCRIPTION
**Pull request by RafaelKa**
This Change adds IDs in for all Editor Placeholders.

### CODE REVIEW

**Comment by RafaelKa** [comment]
Currently no needs to merge, because the placeholders are not translated in Neos.

But i'll tackle in this stuff and make PR in neos-dev...

File: Resources/Private/Translations/en/NodeTypes/OpenGraphMixin.xlf [hunk 1]
```diff
@@ -11,9 +11,15 @@
 			<trans-unit id="properties.openGraphTitle" xml:space="preserve">
 				<source>Open Graph title</source>
 			</trans-unit>
+			<trans-unit id="properties.openGraphTitle.textAreaEditor.placeholder" xml:space="preserve">
+				<source>Used as og:title , max. 60 chars</source>
```
Comment (aertmann) [inline Resources/Private/Translations/en/NodeTypes/OpenGraphMixin.xlf]: why the space before comma?

**Comment by aertmann** [comment]
this could go into a stable release right? so should probably be targeted against the `1.0` branch instead of `master?`

File: Resources/Private/Translations/en/NodeTypes/OpenGraphMixin.xlf (hunk 1 above)
Comment (RafaelKa) [inline Resources/Private/Translations/en/NodeTypes/OpenGraphMixin.xlf]: becaus i copypasted from https://github.com/neos/neos-seo/blob/master/Configuration/NodeTypes.Mixins.yaml#L218

**Comment by RafaelKa** [comment]
Also not sure what to do, but applying that with
https://github.com/RafaelKa/neos-seo/commit/02bce2cbfe61b2c74720092a221326eb1a137717
must be done after Neos can translate the placeholders in JS Part
Currently you'll see only IDs in Inspector not the translations.

So i thought until "applying" the folks from crowdin can translate this strings.

**Comment by aertmann** [comment]
okay, but still should go against `1.0` branch

File: Resources/Private/Translations/en/NodeTypes/OpenGraphMixin.xlf (hunk 1 above)
Comment (aertmann) [inline Resources/Private/Translations/en/NodeTypes/OpenGraphMixin.xlf]: oh okay, well both should be fixed then

**Pull request closed without merging**